src/status_im/ui/screens/wallet/db.cljs

```diff
@@ -24,12 +24,9 @@
   (let [amount' (string/replace amount #"," ".")
         amount-splited (string/split amount' #"[.]")]
     (cond
-      (or (nil? amount) (= amount "") (re-matches #"0[,.]0*$" amount))
+      (or (nil? amount) (= amount "") (= amount "0") (re-matches #"0[,.]0*$" amount))
```

Comment:
Can we name whatever this condition is? Looks like something like `no-amount-present?`, could be in let or if we re-use it.

Not part of PR, but I also noticed there is some weird parse float / try-catch with money just under this piece of code, it seems like that belongs in `money.cljs`?

Reply:
:+1: